llvm/test/MC/RISCV/corev/mac/invalid.s

```diff
@@ -0,0 +1,72 @@
+# RUN: not llvm-mc -triple=riscv32 %s 2>&1 \
+# RUN:        | FileCheck %s --check-prefix=CHECK-ERROR
+
+cv.mac t0, t1, t2
+# CHECK-ERROR: instruction requires the following: 'XCoreVMac' (Multiply-Accumulate)
```

Comment:
Is this capitalisation the canonical way of writing the xcorevmac extension?

Reply:
We used the same formatting for the HWLP instructions. Also other instruction groups are mixed case in the error message, like `'Zvamo'(Vector AMO Operations)`. Though they are not that wild.

Reply:
Indeed, that's something I should have picked up when reviewing that. I think title case is fine, but it's more about aiming for something consistent between the entire tool set. @jeremybennett do you have any better visibility on what any recommended capitalisation of these features?

Reply:
I renamed it to `Xcorevmac`. This is consistent with every other extension name, which are all in title case.

I will rename the rest (HWLP, XCoreV) in a separate PR, if this is the format we should decide to go with.

Reply:
This seems consistent to me.